options/internal/generic/locale.cpp

```diff
@@ -73,6 +73,10 @@ char *nl_langinfo(nl_item item) {
 				__ensure(!"ABDAY_* constants don't seem to be contiguous!");
 				__builtin_unreachable();
 		}
+	}else if(item == D_FMT) {
+		return const_cast<char *>("%m/%d/%y");
+	}else if(item == T_FMT) {
+		return const_cast<char *>("%H:%M:%S");
```

Comment:
This is invalid (you can't edit literals, and it's specified that these should be a static buffer that the caller can edit)

Reply:
This whole function needs to be fixed in that case, as all the above returns also do this pattern.